src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -58,6 +63,32 @@
     (do (utils/log "unknown type: " type)
         value)))
 
+(defn select2-typeahead [placeholder r on-item-selected]
+  (.select2 (js/$ r)
+            (clj->js { :ajax { :url (fn [params]
+                                      (str (config/api-url-root) "/duos/autocomplete/" (aget params "term")))
```

Comment:
add a TODO comment: when we support multiple ontologies, may need a different endpoint.